frontend/src/app/util/text_svg_position.cljs

```diff
@@ -90,7 +90,7 @@
   [shape-id]
   (when (some? shape-id)
     (p/let [text-data (calc-text-node-positions shape-id)]
-      (when (d/not-empty? text-data)
+      (if (d/not-empty? text-data)
```

Comment:
I think we don't need the `if` if `text-data` is empty or nil the `mapv` function will return `[]`

Reply:
Thanks! :)